llvm/include/llvm/BinaryFormat/Dwarf.h

```diff
@@ -613,6 +613,24 @@ enum AcceleratorTable {
   DW_hash_function_djb = 0u
 };
 
+// Uniquify the string hashes and calculate the bucket count for the
+// DWARF v5 Accelerator Table.
+inline uint32_t
+ComputeDebugNamesUniqueHashes(SmallVector<uint32_t, 0> &hashes) {
+  uint32_t BucketCount = 0;
+
+  llvm::sort(hashes);
```

Comment:
AFAICT we are already inside `namespace llvm`. (same comment below)

Reply:
Fixed, thanks.